main/options.c

```diff
@@ -302,6 +312,10 @@ static optionDescription LongOptionDescription [] = {
  {1,"       Output list of language mappings."},
  {1,"  --map-<LANG>=[+]map"},
  {1,"       Alternative version of --langmap option."},
+#ifdef HAVE_ICONV
+ {1,"  --output-encoding=encoding"},
+ {1,"      Specify encoding to write tags file."},
```

Comment:
If only `--output-encoding` is provided, nothing happens at all.  It's relatively confusing, so I'd say that we either need to warn if ``output-encoding| is used without `--input-encoding`, or make `--input-encoding` default to something sensible when `--output-encoding`  is given.